app/uk/gov/hmrc/excisemovementcontrolsystemapi/services/PushNotificationService.scala

```diff
@@ -78,7 +79,13 @@ class PushNotificationServiceImpl @Inject()(
       getArcOrThrowIfEmpty(movement.administrativeReferenceCode, messageId),
       ern)
 
-    notificationConnector.postNotification(boxId, notification)
+    movement.boxId match {
```

Comment:
I would not change this as this will be protected from the feature flag and should never be call when the feature flag is false

Reply:
Scenario is as follows: 
1. Feature flag is false
2. Create movement - so no box Id
3. Feature flag turned on
4. Get message for movement
5. Tries to send push notification to empty box id.

We probably will hit this case anyway in the "PPN is down" story where we don't get a box Id back and carry on.